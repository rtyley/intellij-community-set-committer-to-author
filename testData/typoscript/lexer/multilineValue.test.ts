myObject(
  some value
)

myObject.nextProperty (ignored text
  some other value ))
  and one more string with provocations{
    [GLOBAL]
  }
)

myObject3 (
 prop{
  p1 = 2
  p4 = 5
 }
)also ignored text
