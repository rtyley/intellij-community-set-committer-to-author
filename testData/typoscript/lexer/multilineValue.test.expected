OBJECT_PATH_ENTITY ('myObject')
MULTILINE_OPEN ('(')
WHITE_SPACE ('
  ')
MULTILINE_VALUE ('some value')
WHITE_SPACE ('
')
MULTILINE_CLOSE (')')
WHITE_SPACE ('

')
OBJECT_PATH_ENTITY ('myObject')
OBJECT_PATH_SEPARATOR ('.')
OBJECT_PATH_ENTITY ('nextProperty')
WHITE_SPACE (' ')
MULTILINE_OPEN ('(')
IGNORED_TEXT ('IGNORED_TEXT')
WHITE_SPACE ('
  ')
MULTILINE_VALUE ('some other value ))')
WHITE_SPACE ('
  ')
MULTILINE_VALUE ('and one more string with provocations{')
WHITE_SPACE ('
    ')
MULTILINE_VALUE ('[GLOBAL]')
WHITE_SPACE ('
  ')
MULTILINE_VALUE ('}')
WHITE_SPACE ('
')
MULTILINE_CLOSE (')')
WHITE_SPACE ('

')
OBJECT_PATH_ENTITY ('myObject3')
WHITE_SPACE (' ')
MULTILINE_OPEN ('(')
WHITE_SPACE ('
 ')
MULTILINE_VALUE ('prop{')
WHITE_SPACE ('
  ')
MULTILINE_VALUE ('p1 = 2')
WHITE_SPACE ('
  ')
MULTILINE_VALUE ('p4 = 5')
WHITE_SPACE ('
 ')
MULTILINE_VALUE ('}')
WHITE_SPACE ('
')
MULTILINE_CLOSE (')')
IGNORED_TEXT ('also IGNORED_TEXT')
WHITE_SPACE ('
')